cfme/tests/services/test_request.py

```diff
@@ -71,3 +107,34 @@ def test_services_requester_dropdown_sorting(appliance, generic_catalog_item):
 
     all_options = [option.text for option in view.filter_by.requester.all_options]
     assert ["Administrator", "All"] == all_options
+
+
+@pytest.mark.meta(automates=[1641012])
+@pytest.mark.customer_scenario
+def test_user_view_service_request_details(appliance, generic_catalog_item, user):
+    """
+    Bugzilla:
+        1641012
+
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/6h
+        startsin: 5.11
+        testSteps:
+            1. Create Role,Group with Tag and user
+            2. Order the catalog items with admin
+            3. Login with user
+            4. Go to Services -> Requests -> Request details page
+        expectedResults:
+            1.
+            2.
+            3.
+            4. User should be able to see the provision request
+    """
+    request = ServiceCatalogs(
+        appliance, catalog=generic_catalog_item.catalog, name=generic_catalog_item.name
+    ).order()
+    with user:
+        appliance.server.login(user)
```

Comment:
If you're using `with user`, do you need to use `appliance.server.login(user)`, have you tried without it?

Reply:
Yes right, we can log in without  `appliance.server.login(user)` thanks